command-expansion-server/src/packages/lib/CommandTypeCodegen.ts

```diff
@@ -174,7 +174,12 @@ ${doc}
 function mapArgumentType(argument: ampcs.FswCommandArgument, enumMap: ampcs.EnumMap): string {
   switch (argument.arg_type) {
     case "enum":
-      return enumMap[argument.enum_name].values.map((value) => `'${value.symbol}'`).join(" | ");
+      // boolean enum shouldn't be "TRUE | FALSE" but of `boolean` type
+      if (argument.enum_name.toLocaleLowerCase().includes("boolean")) {
```

Comment:
The `includes()` here worries me that a false-positive could occur.

Reply:
Agreed. We shouldn't do nominal casting here - thats a footgun waiting to go off.

We should explicitly check that the enum has exactly two variants of 'false' and 'true'.

Reply:
Got it. let me change that.